docs/Qualitaetssicherung.tex

```diff
@@ -0,0 +1,212 @@
+\documentclass[parskip=full]{scrartcl}
+\usepackage[top=2.5cm, bottom=2.5cm, left=2.5cm, right=2.5cm]{geometry}
+\usepackage[utf8]{inputenc}
+\usepackage[T1]{fontenc}
+\usepackage[german]{babel}
+\usepackage{hyperref}
+\usepackage[toc, nonumberlist]{glossaries}
+\usepackage{graphicx}
+\usepackage{enumitem}
+\usepackage{float}
+\usepackage{color}
+\usepackage{textcomp}
+
+\hypersetup{
+  colorlinks=false,
+  linktoc=all,
+  hidelinks,
+}
+
+\title{OSIP - Testbericht}
+\subtitle{OPC UA Simulator for Industrial Plants}
+\author{
+    M. Armbruster\\
+    D. Kahles\\
+    H. Lehmann\\
+    M. Schwarzmann\\
+    N. Wilhelm
+}
+
+\begin{document}
+\maketitle
+\thispagestyle{empty}
+\vspace{20px}
+\begin{center}
+  \includegraphics[scale=0.4]{../icon.png}
+\end{center}
+\pagebreak
+\tableofcontents
+\pagebreak
+
+\section{Einleitung}
+
+\section{Änderungen in der Implementierung}
+
+\section{Unittests}
+
+\section{Anforderungen und Tests}
+In Folge unterschiedlicher Umsetzungen von Anforderungen im Vergleich zum Pflichtenheft erfolgt zunächst die Anpassung der Anforderungen und Tests durch die vorhandene Rückkoppelung.
+
+\subsection{Angepasste Anforderungen}
+Die Anforderungen [FA100], [FA110], [FA160], [FA180], [FA200], [FA220], [FA270], [FA310], [FA320], [FA450], [FA500], [FA550], [FA570], [FA590], [FA600], [FA610], [FA650], [FA660], [FA710], [FA720], [FA730], [FA740], [FA820], [NF100], [NF120], [NF130], [D20], [D60], [D70], [D80], [D90], [D100] und [D120] wurden nicht implementiert und konnten daher gestrichen werden.
```

Comment:
Gerade nochmal geschaut. Die stehen da tatsächlich nicht. Zumindest in der Version im Master. Hab jetzt nicht alle nochmal kontrolliert, das hatte ich beim Schreiben vom Kommentar schon gemacht